docs/03-privacy/privacy-incident-response.md

```diff
@@ -0,0 +1,929 @@
+unfairness to an individual.” Examples of Sensitive PII include, but are not limited to:
+
+1. Social Security numbers (SSN)
+1. drivers license or state identification numbers
+1. passport numbers
+1. Alien Registration numbers
+1. financial account numbers
+1. biometric identifiers
+1. Other data, when combined, may also constitute Sensitive PII, such as:
+   1. citizenship or immigration status
+   1. medical information
+   1. salary
+   1. ethnic or religious affiliation
+   1. personal email address, address, and phone
+   1. account passwords
+   1. date of birth
+   1. criminal history
+   1. mother’s maiden name
+
+## Obligations
+
+As an employee, appointee, detailee, intern, contractor, grantee,
+or consultant (hereafter, Bixal personnel), you have an obligation to report suspected or confirmed
+privacy incidents in a timely manner. By reporting a privacy incident in
+a timely manner, Bixal personnel can initiate the privacy incident response process, which is
+required by federal law and policy.
+
+The PICR is an instructional “roadmap” for responding to privacy incidents,
+addressing reporting to resolution of an incident, as well as developing lessons learned. This
+guidance describes the roles and responsibilities of Bixal personnel,
+including employees, supervisors, Component Privacy Officers/Privacy Points of Contact (PPOCs) as
+well as the responsible Security Operations Center (Information Security Office).
+
+All have a critical role at the outset in establishing facts that will be needed,
+not only to contain the privacy incident, but also to identify appropriate mitigations and
+lessons learned.
+
+Privacy incidents, whether accidental or malicious, can pose specific risks to individuals, because
+there is an increasing recognition that personal information such as
+Social Security numbers, financial account information, health information, and biometric data,
+is valuable and can be reverse engineered with a potential for great
+public harm. Therefore, it is crucial that Bixal personnel be able to identify and
+report a suspected or confirmed privacy incident. Taking immediate action to report a
+suspected or confirmed privacy incident is the first step in containing, mitigating, and remediating a privacy
+incident.
+
+## Roles and Responsibilities
+
+### Director of Technology
+
+* Serves as the senior employee responsible for oversight of privacy incident
+management.
+* Responsible for determining whether Bixal's response can be conducted at
+the direction of the Information Security Officer assigned to the component, or
+if the Office of Information Security will lead the response. The Director may choose
+not to convene the BISO if the response can be conducted at the Component level. At a
+minimum, the BISO is convened when a privacy incident constitutes a “major incident,” as
+defined in OMB M-18-0210 and subsequent OMB Guidance.
```

Comment:
May be a good idea here to link the Memorandum: https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2017/M-18-02%20%28final%29.pdf ?